### ISSUE TITLE
Problem Reading Ribbons and its Corresponding Elements with Shortcut Keys in Word 2013

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#5431]
**Reported by netblue on 2015-10-19 20:55**
High, since latest snaps NVDA does not announce the ribbons when using shortcut keys to open them. Also the elements on the ribbons are not red regardless if shortcut keys or the arrow keys are used to find them. This makes using Word 2013 impossible with NVDA. I tested this with Jaws 16 and 15 and all ribbons and elements are announced perfectly.
I am using Windows 7  Ultamate  64 bits.

**Comment by ehollig** [comment]
All ribbons and elements are announced perfectly in Microsoft Word 2016. Can someone confirm that it works in 2013? Otherwise, I would suggest a close.

**Comment by ruifontes** [comment]
Yes, it also works in 2013.

Rui


-----Mensagem Original----- 
De: ehollig
Data: 12 de julho de 2017 16:35
Para: nvaccess/nvda
Cc: Subscribed
Assunto: Re: [nvaccess/nvda] Problem Reading Ribbons and its Corresponding 
Elements with Shortcut Keys in Word 2013 (#5431)



All ribbons and elements are announced perfectly in Microsoft Word 2016. Can 
someone confirm that it works in 2013? Otherwise, I would suggest a close.

**Comment by jcsteh** [comment]
This was possibly a regression introduced during the development of #3831. I haven't seen any further reports of this either. Closing.
